File: app/src/main/java/com/live/utils/MaterialDialogUtil.java
package com.live.utils;

import android.app.Activity;
import android.content.DialogInterface;

import com.afollestad.materialdialogs.MaterialDialog;

public class MaterialDialogUtil {

    public static void showMsgWithoutTitle(final Activity activity, String content, String positiveLabel, MaterialDialog.SingleButtonCallback callback) {

        new MaterialDialog.Builder(activity).content(content).positiveText(positiveLabel).onPositive(callback)
                .cancelListener(new DialogInterface.OnCancelListener() {
                    @Override
                    public void onCancel(DialogInterface dialog) {
                        activity.finish();
                    }
                }).build().show();
    }

    public static void showMsgWithoutTitle(Activity activity, String content, String positiveLabel,String negativeLabel,
                                           MaterialDialog.SingleButtonCallback positiveCallback, MaterialDialog.SingleButtonCallback negativeCallback) {
        new MaterialDialog.Builder(activity).content(content).positiveText(positiveLabel).onPositive(positiveCallback)
                .negativeText(negativeLabel).onNegative(negativeCallback).build().show();
    }


}

File: app/src/main/java/com/live/utils/MaterialDialogUtil.kt
package com.live.utils

import android.app.Activity
import com.afollestad.materialdialogs.DialogCallback

import com.afollestad.materialdialogs.MaterialDialog

object MaterialDialogUtil {

    fun showMsgWithoutTitle(activity: Activity, content: String, positiveLabel: String, callback: DialogCallback) {
        MaterialDialog(activity).message(null, content).positiveButton(null, positiveLabel, callback).negativeButton { activity.finish() }.show()
    }

}
